templates/user/dashboard/issues.tmpl

```diff
@@ -126,6 +126,10 @@
 										<span class="octicon octicon-checklist"></span> {{$tasksDone}} / {{$tasks}} <span class="progress-bar"><span class="progress" style="width:calc(100% * {{$tasksDone}} / {{$tasks}});"></span></span>
 									</span>
 								{{end}}
+								{{if ne .DeadlineUnix 0}}
+									<span class="octicon octicon-calendar"></span>
+									<span{{if .IsOverdue}} class="overdue"{{end}}>{{.DeadlineUnix.FormatShort}}</span>
```

Comment:
somehow if i apply this patch the ti shows me the "Time Tracker" value!?!

think there is something wrong in the go code :(